projects/demo-integrations/cypress/support/close-preview.ts

```diff
@@ -0,0 +1,3 @@
+export const tuiClosePreview = (): void => {
+    cy.get(`tui-preview`).find(`button[icon=tuiIconCloseLarge]`).click();
```

Comment:
I think it is overkill)

https://docs.cypress.io/api/cypress-api/custom-commands#1-Don-t-make-everything-a-custom-command

Reply:
fixed